Documentation/proposals/202409-instance-status-subresource.md

```diff
@@ -1,11 +1,11 @@
-# Status subresource for Prometheus operator CRDs
+# Status subresource for Prometheus operator Instance CRDs
```

Comment:
Maybe let's have another PR to refine this page and scope it to "workload" CRDs only?